### DISCUSSION TITLE
In the new RBAC, how the Is creator works in the Author role?

### DISCUSSION THREAD

**Discussion by abdonrd** [strapi/strapi#7156] [Legacy Topics]
In the new RBAC, how the `Is creator` works in the `Author` role?

<img width="1046" alt="Screenshot 2020-07-24 at 12 46 59" src="https://user-images.githubusercontent.com/1007051/88383988-da12fa80-cdab-11ea-96a7-c5f14e7c6d5e.png">

<img width="942" alt="Screenshot 2020-07-24 at 12 47 24" src="https://user-images.githubusercontent.com/1007051/88383994-dc755480-cdab-11ea-8052-d09cf0132694.png">

**Comment by lauriejim** [comment]
Hello! All this logic is managed by the `findEntityAndCheckPermissions` in the Content Manager.
To answer your question, the user of this role will be able to delete only Links that have been created by him.

**Comment by abdonrd** [comment]
@lauriejim thanks Jim! But how Strapi knows if have been created by him?

**Reply by lauriejim** [reply]
A relation is automatically created (like the `created_at` and `updated_at`)
A field `created_by` exist

**Reply by abdonrd** [reply]
And what about all the information that was created before `v3.1.0`?

**Reply by lauriejim** [reply]
That's a good question.
@alexandrebodin any insight on that?

**Reply by alexandrebodin** [reply]
Data that don't have a creator can be edited by editors and admin not authors as an author can only edit its own content :)

**Reply by alexandrebodin** [reply]
@abdonrd You could always create migration in DB (update to set the created_by id)  if you really need to but we decided that there was no right choice depending on user needs.

**Reply by abdonrd** [reply]
Thanks both!

**Reply by lauriejim** [reply]
Thank you for the info @alexandrebodin

**Comment by derrickmehaffy** [comment]
Locking as migrated to: https://forum.strapi.io/t/in-the-new-rbac-how-the-is-creator-works-in-the-author-role-7156/1540
